browser-tests/functional-tests/functional.spec.js

```diff
@@ -107,6 +107,33 @@ test.describe("Android tests", () => {
   });
 });
 
+test.describe("Error tests", () => {
+  test("Handles an unexpected error from core-back", async ({ page }) => {
+    // Start a session
+    await page.goto(getAuthoriseUrlForJourney("testError"));
+
+    // Go to the error
+    await page.goto("/ipv/journey/page-ipv-identity-document-start/error")
+
+    // When we come back with an error
+    const textLocator = await page.getByText("Sorry, there is a problem");
+    await expect(textLocator).toBeVisible();
+  });
+
+  test("Handles an unexpected error after a CRI callback", async ({ page }) => {
```

Comment:
This is the test that fails without the import change - curiously it only seems to happen when we get an error on the follow-up journey engine request (not the cri callback itself).